src/main/java/com/nbe8101team03/domain/orders/service/OrderService.java

```diff
@@ -0,0 +1,108 @@
+package com.nbe8101team03.domain.orders.service;
+
+import com.nbe8101team03.domain.orders.dto.OrderResponse;
+import com.nbe8101team03.domain.orders.entity.Order;
+import com.nbe8101team03.domain.orders.entity.OrderStatus;
+import com.nbe8101team03.domain.orders.repository.OrderRepository;
+import com.nbe8101team03.domain.product.entity.Product;
+import com.nbe8101team03.domain.product.repository.ProductRepository;
+import com.nbe8101team03.domain.user.entity.User;
+import com.nbe8101team03.domain.user.repository.UserRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+@Transactional
+public class OrderService {
+
+    private final OrderRepository orderRepository;
+    private final UserRepository userRepository;
+    private final ProductRepository productRepository;
+
+
+//    주문하기
+    public OrderResponse createOrder(String email, String address, int zipcode, Long productId , int quantity) {
+//       유저생성
+        User user = userRepository.findByEmail(email)
+                .orElseGet(() -> {
+                    User newUser = new User();
+                    newUser.setEmail(email);
+                    newUser.setAddress(address);
+                    newUser.setZipcode(zipcode);
+                    return userRepository.save(newUser);
+                });
+
+
+//        상품 조회
+        Product product = productRepository.findById(productId).orElseThrow(() -> new IllegalArgumentException("상품이 존재하지 않습니다. productId=" + productId));
```

Comment:
공통 custom exception 사용에 따르면
```suggestion
        Product product = productRepository.findById(productId).orElseThrow(() -> new ProductException(ProductErrorCode.UNKNWON_PRODUCT, "상품이 존재하지 않습니다. productId=" + productId));
```
다음과 같이 수정하여 사용하는 것을 권장 드립니다.